### DISCUSSION TITLE
v1.68.0

### DISCUSSION THREAD

**Discussion by alextran1502** [immich-app/immich#3345] [Announcements]
# v1.68.0

## Highlights
Hello, Immich fans! Welcome to the `v1.68.0` release of Immich. There are many QoS improvements in this version that you'll like. Some of the key highlights include:

- Add the ability to hide and show faces
- Better user interface to manage face’s name and facial recognition action
- Add shortcuts for common actions to photo viewer on the web

### Hide and show faces

You can now select irrelevant faces to hide them. The hidden faces won’t be displayed in search results and the people section in the info panel.

https://github.com/immich-app/immich/assets/27055614/af388a29-958e-4cf4-8319-aa8bc2759197

### Better face management options

You can now use a context menu on the people page to add/change a person's name without going into the detailed view of the person. You can also initiate the face-merging process from this menu.

[<<insert-video-here>>
](https://github.com/immich-app/immich/assets/27055614/7a4c2851-9ef4-4261-a116-f872e0a9fd10)
### Add keyboard shortcuts to common actions.

Some of us only navigate the world and the web with a keyboard (looking at you, Vim and Emacs users). So it would take away the sacred weapon of choice by introducing many clicks to perform repetitive actions. So we add quick shortcuts for the following action on the web.

#### In the detail view
- `shift + D`:  Download a photo or video
- `f`: to toggle favorite status
- `shift + A`: to archive or unarchive
- `del`: to delete an asset
- `i`: To toggle the info panel

#### In the main timeline
`/` to navigate to the Explore page

- - - -

And as always, bugs are fixed, and many other improvements also come with this release.

Please consider supporting the project.

## Support

<p align="center">
<img src="https://media.giphy.com/media/LStqgGESXW8XnuCv5y/giphy.gif" width="250" title="Loading ~4000 images/videos"> 
</p>

If you find the project helpful, you can support Immich via the following channels.

- Monthly donation via [GitHub Sponsors](https://github.com/sponsors/alextran1502)
- One-time donation via [GitHub Sponsors](https://github.com/sponsors/alextran1502?frequency=one-time&sponsor=alextran1502)
- [Librepay](https://liberapay.com/alex.tran1502/)
- [buymeacoffee](https://www.buymeacoffee.com/altran1502)
- Bitcoin: 1FvEp6P6NM8EZEkpGUFAN2LqJ1gxusNxZX

It is a great way to let me know that you want me to continue developing and working on this project for years to come.

🎉 Cheers! 🎉

### Web
* feat(web): show download size by @jrasm91 in https://github.com/immich-app/immich/pull/3270
* fix(web): update style of rows in user administration table by @tranquanghuy0801 in https://github.com/immich-app/immich/pull/3277
* chore(web): Update to Svelte 4 by @alextran1502 in https://github.com/immich-app/immich/pull/3196
* fix(web): navigation buttons z-order by @brighteyed in https://github.com/immich-app/immich/pull/3286
* feat(web): set asset as profile picture by @faupau03 in https://github.com/immich-app/immich/pull/3106
* fix(web): Scroll to top fix by @KailashGanesh in https://github.com/immich-app/immich/pull/3302
* feat(web): new shortcuts by @martabal in https://github.com/immich-app/immich/pull/3111
* fix(web): remove dependency on rxjs  by @uhthomas in https://github.com/immich-app/immich/pull/3301
* fix(web): allow edit to empty name by @martabal in https://github.com/immich-app/immich/pull/3322
* feat(web): add better face management UI action by @alextran1502 in https://github.com/immich-app/immich/pull/3328
* chore(web): sort tailwindcss class automatically by @alextran1502 in https://github.com/immich-app/immich/pull/3330

### Server
* feat(server,web): hide faces by @martabal in https://github.com/immich-app/immich/pull/3262
* refactor(server): upload config by @jrasm91 in https://github.com/immich-app/immich/pull/3252
* refactor(server): auth/oauth by @jrasm91 in https://github.com/immich-app/immich/pull/3242
* chore(server): add ability to use docker secrets file for REDIS_PASSWORD by @abhi-chakrab in https://github.com/immich-app/immich/pull/3282
* feat(server): storage label claim by @jrasm91 in https://github.com/immich-app/immich/pull/3278
* feat(server): add endpoint to get supported media types on the server by @alextran1502 in https://github.com/immich-app/immich/pull/3284
* chore(server): adjust loglevel of reverse geocoding intializer to LOG by @oddlama in https://github.com/immich-app/immich/pull/3303

### Mobile
* chore(mobile): share logo platform consistency by @alextran1502 in https://github.com/immich-app/immich/pull/3290
* fix(mobile): fix forgetting backup albums (#3108) by @madadam in https://github.com/immich-app/immich/pull/3244
* chore(mobile): remove things sections by @alextran1502 in https://github.com/immich-app/immich/pull/3309
* fix(mobile): Allow sign out if server is down, or device is offline by @ddshd in https://github.com/immich-app/immich/pull/3275

### Documentation and others
* docs: Remove a duplicate word by @Skkay in https://github.com/immich-app/immich/pull/3285
* docs: Update _storage-template.md by @briandipalma in https://github.com/immich-app/immich/pull/3291
* chore: Enable logging on typesense container by @bo0tzz in https://github.com/immich-app/immich/pull/3326

## New Contributors
* @Skkay made their first contribution in https://github.com/immich-app/immich/pull/3285
* @abhi-chakrab made their first contribution in https://github.com/immich-app/immich/pull/3282
* @briandipalma made their first contribution in https://github.com/immich-app/immich/pull/3291
* @KailashGanesh made their first contribution in https://github.com/immich-app/immich/pull/3302
* @oddlama made their first contribution in https://github.com/immich-app/immich/pull/3303
* @madadam made their first contribution in https://github.com/immich-app/immich/pull/3244

**Full Changelog**: https://github.com/immich-app/immich/compare/v1.67.2...v1.68.0

<hr /><em>This discussion was created from the release <a href='https://github.com/immich-app/immich/releases/tag/v1.68.0'>v1.68.0</a>.</em>

**Comment by Shakri007** [comment]
> # v1.68.0
> 
> ## Highlights
> Hello, Immich fans! Welcome to the `v1.68.0` release of Immich. There are many QoS improvements in this version that you'll like. Some of the key highlights include:
> 
> - Add the ability to hide and show faces
> - Better user interface to manage face’s name and facial recognition action
> - Add shortcuts for common actions to photo viewer on the web
> 
> ### Hide and show faces
> 
> You can now select irrelevant faces to hide them. The hidden faces won’t be displayed in search results and the people section in the info panel.
> 
> 
> https://github.com/immich-app/immich/assets/27055614/af388a29-958e-4cf4-8319-aa8bc2759197
> 
> 
> ### Better face management options
> 
> You can now use a context menu on the people page to add/change a person's name without going into the detailed view of the person. You can also initiate the face-merging process from this menu.
> 
> [<<insert-video-here>>
> ](https://github.com/immich-app/immich/assets/27055614/7a4c2851-9ef4-4261-a116-f872e0a9fd10)
> ### Add keyboard shortcuts to common actions.
> 
> Some of us only navigate the world and the web with a keyboard (looking at you, Vim and Emacs users). So it would take away the sacred weapon of choice by introducing many clicks to perform repetitive actions. So we add quick shortcuts for the following action on the web.
> 
> #### In the detail view
> - `shift + D`:  Download a photo or video
> - `f`: to toggle favorite status
> - `shift + A`: to archive or unarchive
> - `del`: to delete an asset
> - `i`: To toggle the info panel
> 
> #### In the main timeline
> `/` to navigate to the Explore page
> 
> - - - -
> 
> And as always, bugs are fixed, and many other improvements also come with this release.
> 
> Please consider supporting the project.
> 
> ## Support
> 
> <p align="center">
> <img src="https://media.giphy.com/media/LStqgGESXW8XnuCv5y/giphy.gif" width="250" title="Loading ~4000 images/videos"> 
> </p>
> 
> 
> If you find the project helpful, you can support Immich via the following channels.
> 
> - Monthly donation via [GitHub Sponsors](https://github.com/sponsors/alextran1502)
> - One-time donation via [GitHub Sponsors](https://github.com/sponsors/alextran1502?frequency=one-time&sponsor=alextran1502)
> - [Librepay](https://liberapay.com/alex.tran1502/)
> - [buymeacoffee](https://www.buymeacoffee.com/altran1502)
> - Bitcoin: 1FvEp6P6NM8EZEkpGUFAN2LqJ1gxusNxZX
> 
> 
> It is a great way to let me know that you want me to continue developing and working on this project for years to come.
> 
> 🎉 Cheers! 🎉
> 
> 
> ## Changelogs
> 
> ### Web
> * feat(web): show download size by @jrasm91 in https://github.com/immich-app/immich/pull/3270
> * fix(web): update style of rows in user administration table by @tranquanghuy0801 in https://github.com/immich-app/immich/pull/3277
> * chore(web): Update to Svelte 4 by @alextran1502 in https://github.com/immich-app/immich/pull/3196
> * fix(web): navigation buttons z-order by @brighteyed in https://github.com/immich-app/immich/pull/3286
> * feat(web): set asset as profile picture by @faupau03 in https://github.com/immich-app/immich/pull/3106
> * fix(web): Scroll to top fix by @KailashGanesh in https://github.com/immich-app/immich/pull/3302
> * feat(web): new shortcuts by @martabal in https://github.com/immich-app/immich/pull/3111
> * fix(web): remove dependency on rxjs  by @uhthomas in https://github.com/immich-app/immich/pull/3301
> * fix(web): allow edit to empty name by @martabal in https://github.com/immich-app/immich/pull/3322
> * feat(web): add better face management UI action by @alextran1502 in https://github.com/immich-app/immich/pull/3328
> * chore(web): sort tailwindcss class automatically by @alextran1502 in https://github.com/immich-app/immich/pull/3330
> 
> ### Server
> * feat(server,web): hide faces by @martabal in https://github.com/immich-app/immich/pull/3262
> * refactor(server): upload config by @jrasm91 in https://github.com/immich-app/immich/pull/3252
> * refactor(server): auth/oauth by @jrasm91 in https://github.com/immich-app/immich/pull/3242
> * chore(server): add ability to use docker secrets file for REDIS_PASSWORD by @abhi-chakrab in https://github.com/immich-app/immich/pull/3282
> * feat(server): storage label claim by @jrasm91 in https://github.com/immich-app/immich/pull/3278
> * feat(server): add endpoint to get supported media types on the server by @alextran1502 in https://github.com/immich-app/immich/pull/3284
> * chore(server): adjust loglevel of reverse geocoding intializer to LOG by @oddlama in https://github.com/immich-app/immich/pull/3303
> 
> ### Mobile
> * chore(mobile): share logo platform consistency by @alextran1502 in https://github.com/immich-app/immich/pull/3290
> * fix(mobile): fix forgetting backup albums (#3108) by @madadam in https://github.com/immich-app/immich/pull/3244
> * chore(mobile): remove things sections by @alextran1502 in https://github.com/immich-app/immich/pull/3309
> * fix(mobile): Allow sign out if server is down, or device is offline by @ddshd in https://github.com/immich-app/immich/pull/3275
> 
> ### Documentation and others
> * docs: Remove a duplicate word by @Skkay in https://github.com/immich-app/immich/pull/3285
> * docs: Update _storage-template.md by @briandipalma in https://github.com/immich-app/immich/pull/3291
> * chore: Enable logging on typesense container by @bo0tzz in https://github.com/immich-app/immich/pull/3326
> 
> 
> ## New Contributors
> * @Skkay made their first contribution in https://github.com/immich-app/immich/pull/3285
> * @abhi-chakrab made their first contribution in https://github.com/immich-app/immich/pull/3282
> * @briandipalma made their first contribution in https://github.com/immich-app/immich/pull/3291
> * @KailashGanesh made their first contribution in https://github.com/immich-app/immich/pull/3302
> * @oddlama made their first contribution in https://github.com/immich-app/immich/pull/3303
> * @madadam made their first contribution in https://github.com/immich-app/immich/pull/3244
> 
> **Full Changelog**: https://github.com/immich-app/immich/compare/v1.67.2...v1.68.0
> 
> 
> 
> <hr /><em>This discussion was created from the release <a href='https://github.com/immich-app/immich/releases/tag/v1.68.0'>v1.68.0</a>.</em>
